models/models.go

```diff
@@ -69,14 +73,39 @@ type UserData struct {
 	Key   []byte
 }
 
-func CreateUser(email string) (utils.User, error) {
-	user := utils.User{Email: email}
+/*
+ * LoadRandomUser The Let's Encrypt v1 acme API has shut down user creation to force users to adopt v2.
+ * In an attempt to contiune using v1 while we develop a v2 compliant broker, we are replacing
+ * calls to create a new user for each new domain registration with a method that fetches an existing user
+ * from a pool of ids. The random selection of users from a pool aims to minimize the impact of the following rate limits:
+ *	- 300 Pending Authorizations per account
+ *	- Failed Validation limit of 5 failures per account, per hostname, per hour.
+ */
+func LoadRandomUser(db *gorm.DB, userIDPool []string) (utils.User, error) {
+	var user utils.User
+	defer func() {
+		if r := recover(); r != nil {
```

Comment:
What's this bit?  The rest lgtm.

Reply:
`rand.RandN` can sometimes panic, this just recovers from the panic and prevents the runtime from crashing.

Reply:
Interesting.  Wouldn't we want the process to die and restart in that case?

Reply:
(also, do we know why it panics?)

Reply:
@tammersaleh you ok letting this flow on as it is a temp fix?